### PR TITLE
Add events property

### PR DESCRIPTION
**Pull request by dpgraham**
COPY OF https://github.com/appium/python-client/pull/427

Needed to run python isort

### CODE REVIEW

**Comment by ki4070ma** [comment]
@dpgraham 
I've checked in my local so that `webdriver_test.py` needs fix.
(At least, `python -m isort -rc .` works in my local)

Please run below command.

```
$python -m isort -rc test/unit/webdriver/webdriver_test.py
```

**Comment by dpgraham** [comment]
@ki4070ma I tried that already and it still fails the test

**Comment by ki4070ma** [comment]
@dpgraham 
Add `mock = "~=3.0"` to `Pipfile`.

https://github.com/appium/python-client/blob/master/Pipfile

https://pypi.org/project/mock/

Error log from Travis
```
Traceback:

test/unit/webdriver/webdriver_test.py:22: in <module>

    from mock import patch

E   ImportError: No module named mock
```

File: Pipfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ verify_ssl = true
 
 [dev-packages]
 pre-commit = "~=1.13"
+mock = "~=3.0"
```
Comment (ki4070ma) [inline Pipfile]: Need to be under `[packages]`.

(Would like to re-consider which modules should be for dev-packages, packages)

**Comment by KazuCocoa** [comment]
https://github.com/appium/python-client/commit/9e46aa117439341d0b25ced4fd7f9a9bcb7e4a10
mm, my change does not appear here.

`isort` seems to depend on environment.
By hand and by pre-commit hook on my end got a different result.
The former addressed https://github.com/appium/python-client/commit/9e46aa117439341d0b25ced4fd7f9a9bcb7e4a10 , but the later had no issue without the change.

**Comment by ki4070ma** [comment]
@KazuCocoa 
Sorry for confusing you, here change(https://github.com/appium/python-client/pull/429/commits/f2a6abdbe09aad8daa6b31423ca19d48421a3f06) is from github.com directly, so pre-commit doesn't run.

**Comment by KazuCocoa** [comment]
No.
I meant the result of `isort` by hand, `python -m isort -rc .`,  and by pre-commit had a different result on my local.
We should consider if we continue to use it or fix the dependency.

**Comment by KazuCocoa** [comment]
https://pypi.org/project/Appium-Python-Client/0.47/
cc @vrunoa

**Pull request merged**